scripts/setup/create_multinode.sh

```diff
@@ -111,20 +111,11 @@ function setup_master() {
         LOGIN_TOKEN="$(server_exec "$MASTER_NODE" 'tmux show-buffer -b token | grep -B 3 "All nodes need to be joined"')"
         echo "$LOGIN_TOKEN"
     done
```

Comment:
This loop need to be redone. It is not robust to wait for the output of phrase 'All nodes need to be joined'. We should wait for the file masterKey to be written instead.